e2e/BUILD.bazel

```diff
@@ -178,3 +178,8 @@ e2e_integration_test(
     # TODO: figure out why this fails on Windows
     tags = ["no-bazelci-windows"],
 )
+
+e2e_integration_test(
+    name = "linker_disabled_regression",
+    bazel_commands = ["run //subdir:binary"],
```

Comment:
nit: should this be two-element argv?

Reply:
Nope, all usages I see have a full command per string.